controller/src/main/java/io/pravega/controller/metrics/StreamMetrics.java

```diff
@@ -92,6 +104,19 @@ public final class StreamMetrics extends AbstractControllerMetrics {
 
     private final OpStatsLogger createScopeLatency;
     private final OpStatsLogger deleteScopeLatency;
+    // Controller EventProcessor events latency
+    private final OpStatsLogger controllerEventProcessorDeleteStreamLatency;
+    private final OpStatsLogger controllerEventProcessorUpdateStreamLatency;
+    private final OpStatsLogger controllerEventProcessorSealStreamLatency;
+    private final OpStatsLogger controllerEventProcessorTruncateStreamLatency;
+    private final OpStatsLogger controllerEventProcessorAutoScaleStreamLatency;
+    private final OpStatsLogger controllerEventProcessorCreateReaderGroupStreamLatency;
```

Comment:
I will prefer to have a name controllerEventProcessorCreateReaderGroupLatency instead of controllerEventProcessorCreateReaderGroupStreamLatency. Same for controllerEventProcessorDeleteScopeStreamLatency, controllerEventProcessorUpdateReaderGroupStreamLatency, controllerEventProcessorDeleteReaderGroupStreamLatency.

Reply:
fixed